lib/appcontroller_client.py

```diff
@@ -367,6 +375,151 @@ def set_deployment_id(self, deployment_id):
     Returns:
       A boolean indicating whether the deployment ID is stored or not.
     """
-    return self.run_with_timeout(10, 'Set deployment ID request timed out.',
-      self.DEFAULT_NUM_RETRIES, self.server.set_deployment_id, self.secret,
-      deployment_id)
+    return self.run_with_timeout(self.DEFAULT_TIMEOUT, 'Set deployment ID request timed out.',
+      self.DEFAULT_NUM_RETRIES, self.server.set_deployment_id, self.secret, deployment_id)
+
+
+  def get_all_stats(self):
+    """ Queries the AppController for all the stats.
+
+    Returns:
+      A hash in string format containing system and platform stats for this node.
```

Comment:
I would change `hash in string format` to `JSON string` since that meaning for hash is specific to ruby.